README.md

```diff
@@ -1,23 +1,36 @@
-Puppet PowerShell DSC Module
-============================
+# dsc
+[wmf-5.0]: https://www.microsoft.com/en-us/download/details.aspx?id=46889
+[DSCResources]: https://github.com/powershell/DSCResources
 
-## Overview
 
-Puppet module for managing Windows poweshell DSC resources.
+#### Table of Contents
+1. [Module Description - What is the dsc module and what does it do](#module-description)
+2. [Prerequisites](#windows-system-prerequisites)
+3. [Setup](#setup)
+4. [Usage](#usage)
+  * [Install Website Example](#install-website-example)
+5. [Limitations](#limitations)
+  * [Known Issues](#known-issues)
+6. [Notes](#notes)
+7. [License](#license)
 
-This module generates Puppet Types based on DSC resources MOF schema files.
+## Module Description
+
+Puppet module for managing Windows PowerShell DSC (Desired State Configuration) resources.
+
+This module generates Puppet Types based on DSC resources' MOF (Managed Object Format) schema files.
 
 In this version, the following DSC Resources are already built and ready for use:
-- All base DSC resources found in PowerShell 5. ([WMF 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889))
-- All DSC resources found in the [Microsoft PowerShell DSC Resource Kit](https://github.com/powershell/DSCResources).
+- All base DSC resources found in PowerShell 5. ([WMF 5.0][wmf-5.0])
+- All DSC resources found in the [Microsoft PowerShell DSC Resource Kit][DSCResources]
 
 This module is available on the [Puppet Forge](https://forge.puppetlabs.com/puppetlabs/dsc)
 
-## Windows systems prerequisites
+## Windows system prerequisites
 
- - PowerShell 5 which is included in the [Windows Management Framework 5.0](https://www.microsoft.com/en-us/download/details.aspx?id=46889). PowerShell v5 is currently in limited preview, so the above link may change after official release.
+ - PowerShell 5 which is included in the [Windows Management Framework 5.0][wmf-5.0]. PowerShell v5 is currently in limited preview, so the above link may change after official release.
```

Comment:
Either of these sounds better

PowerShell 5 which is included in the Windows Management Framework 5.0 release

or

PowerShell 5 which is included in Windows Management Framework 5.0.